.github/workflows/test.yml

```
@@ -37,13 +56,28 @@ jobs:
+      - name: Get logs from node5
+        run: docker logs node5
+
+      - name: Get logs from redis-server-2
+        run: docker logs redis-server-2
       - name: Run Tests
         run: poetry run pytest --cov --cov-report=xml
-
+          
       - name: Upload coverage
         uses: codecov/codecov-action@v5
         if: matrix.python == '3.10' && matrix.platform != 'macos-11'
```

actionlint (1.7.4)

property "platform" is not defined in object type {python: number}

(expression)